### PR TITLE
Update travis config and rosinstall for noetic

### PR DESCRIPTION
**Pull request by bmagyar**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-      - gennaro.raiola@gmail.com
     on_success: change #[always|never|change] # default: change
     on_failure: change #[always|never|change] # default: always
 
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=ros_control.rosinstall
-    - ROS_PARALLEL_TEST_JOBS=-j1
+    - UPSTREAM_WORKSPACE=ros_control.rosinstall
```
Comment (mathias-luedtke) [inline .travis.yml]:
```suggestion
    - UPSTREAM_WORKSPACE='ros_control.rosinstall -ros_control'
```
To prevent from buliding `ros_control` twice.

Comment (bmagyar) [reply .travis.yml]: thanks!

File: .travis.yml [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=ros_control.rosinstall
-    - ROS_PARALLEL_TEST_JOBS=-j1
+    - UPSTREAM_WORKSPACE='ros_control.rosinstall -ros_control'
   matrix:
-    - ROS_DISTRO=melodic ROS_REPO=ros
-    - ROS_DISTRO=melodic ROS_REPO=ros-testing
+    - ROS_DISTRO=noetic ROS_REPO=ros
```
Comment (mathias-luedtke) [inline .travis.yml]: The stable repository does contain ROS packages yet: http://repositories.ros.org/status_page/ros_noetic_default.html

**Comment by bmagyar** [comment]
@ipa-mdl @matthew-reynolds I had to juggle a bunch to get this to pass. The Python2-Python3 switch broke string comparisons in one of the tests I had to chase and it also seems that rostests run from install space which means we have to install most of the test-related stuff.

**Comment by bmagyar** [comment]
I'll go ahead with the merge as thsi is now blocking a bunch of other PR-s. Happy to accomodate any changes post-merge

**Comment by mikaelarguedas** [comment]
This PR seems to have broken deb install on Noetic as now the package installs gtest in a common location that is conflicting with other ROS packages doing the same:
```
Unpacking ros-noetic-controller-manager-tests (0.19.4-1focal.20201207.152344) ...
dpkg: error processing archive /tmp/apt-dpkg-install-2E6ylt/1076-ros-noetic-controller-manager-tests_0.19.4-1focal.20201207.152344_amd64.deb (--unpack):
 trying to overwrite '/opt/ros/noetic/lib/python3/dist-packages/libgtest.so', which is also in package ros-noetic-tf 1.13.2-1focal.20210112.083534
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
```

(I just tried to install ros-noetic-* and am not relying of the deb of this package)

**Comment by matthew-reynolds** [comment]
@mikaelarguedas I think this is resolved by https://github.com/ros/catkin/pull/1135 which has finally been merged. Let me know if not and I'll look into if we're doing anything weird here, but I think our CMakeLists are pretty standard.

**Comment by mikaelarguedas** [comment]
That makes sense, thanks for the pointer 👍

**Pull request merged**